### PR TITLE
Upgraded Vendure Popularity Score plugin to vendure 2.0

### PR DESCRIPTION
**Pull request by dalyathan**
# Description

The changes should make [Vendure Popularity Score plugin](https://github.com/Pinelab-studio/pinelab-vendure-plugins/tree/master/packages/vendure-plugin-popularity-scores) compatibe with vendure 2.0. Should close https://github.com/Pinelab-studio/pinelab-vendure-plugins/issues/217

# Breaking changes

This PR won't introduce any breaking changes.

# Checklist

Please make sure you've done the following checks:

- [X] I have set a clear title
- [X] My PR is small and contains only a single feature. (Split into multiple PR's if needed)
- [X] I have added or updated test cases for important functionality
- [X] I have updated the README if needed
- [X] I have reviewed my own PR
- [X] I have fixed all typo's and have removed unused or commented code

### CODE REVIEW

File: packages/vendure-plugin-popularity-scores/src/sort.service.ts [hunk 1]
```diff
@@ -85,7 +83,7 @@ export class SortService implements OnModuleInit {
       .addGroupBy('product.id')
       .addOrderBy('count', 'DESC')
       .getRawMany();
-    const maxCount = groupedOrderItems?.[0]?.count;
+    const maxCount = groupedOrderLines?.[0]?.count;
```
Comment (martijnvdbrug) [inline packages/vendure-plugin-popularity-scores/src/sort.service.ts:86]: I think we need a modification for this to work properly:

In v1 we could count the orderItems, which is correct, but here (in v2) we are currently counting order lines. But order lines can have different quantities, so we should take the sum of all `orderlines.quantity` correct?

Or, am I missing something? (Tests are succeeding, but I am not sure if the tests cover this)

**Review by martijnvdbrug — CHANGES_REQUESTED** [review]
nice, small remark regarding orderItem vs OrderLine

File: packages/vendure-plugin-popularity-scores/src/sort.service.ts (hunk 1 above)
Comment (dalyathan) [reply packages/vendure-plugin-popularity-scores/src/sort.service.ts:86]: Yeah you're right. I will look into it

**Review by martijnvdbrug — APPROVED** [review]
Great work on extending the testcases

**Pull request merged**